widget/text.go

```diff
@@ -192,6 +200,7 @@ type textRenderer struct {
 	objects []fyne.CanvasObject
 
 	texts []*canvas.Text
+	tLock sync.RWMutex
```

Comment:
I don’t know what “tLock” means - is should it just be “lock” or the more specific “textLock”?